hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestHoodieSanity.java

```diff
@@ -152,10 +152,10 @@ public void testRunHoodieJavaApp(String hiveTableName, String tableType, Partiti
     Assert.assertEquals("Expecting 100 rows to be present in the new table", 80,
         Integer.parseInt(stdOutErr.getLeft().trim()));
 
-    // If is MOR table, ensure realtime table row count is 100 (without duplicates)
+    // If is MOR table, ensure realtime table row count is 100 - 20 = 80 (without duplicates)
     if (tableType.equals(HoodieTableType.MERGE_ON_READ.name())) {
       stdOutErr = executeHiveCommand("select count(1) from " + hiveTableName + "_rt");
-      Assert.assertEquals("Expecting 100 rows to be present in the realtime table,", 100,
+      Assert.assertEquals("Expecting 100 rows to be present in the realtime table,", 80,
```

Comment:
Why is the expected counts changing  ?

Reply:
@nsivabalan : ^^

Reply:
Bcoz, in HoodieJavaApp, I added a delete batch which deletes 20 records.